src/components/TableCreator/hooks/useTableOptions/useTableOptions.ts

```diff
@@ -0,0 +1,29 @@
+import { RowData } from "@tanstack/react-table";
+import { useConfig } from "../../../../hooks/useConfig";
+import { useExpandedOptions } from "../../../Table/options/expanded/hook";
+import { useGroupingOptions } from "../../../Table/options/grouping/hook";
+import { useSortingOptions } from "../../../Table/options/sorting/hook";
+import { UseTableOptions } from "./types";
+
+export function useTableOptions<T extends RowData>(): UseTableOptions<T> {
```

Comment:
New hook to centralise the entity list table options!

The goal here is to pass these options into the `Table` component... so that the `Table` component can be used by either entity list or entity related tables.